config.py

```diff
@@ -82,6 +82,15 @@ class Config(object):
         'g-cloud-5': 'g-cloud-6',
     }
 
+    # Note that because our GTM auth token is rendered into the client environment (see
+    # _javascripts.html), there isn't any point in encrypting it, as we would with other
+    # similar tokens.
```

Comment:
Which is good because I don't think there's a mechanism to rotate them, at least I didn't see it.

@samuelhwilliams said he'd been reading some things about GTM spoofing/security so we'll discuss those before this lands.